src/Presenter/Order/OrderPresenter.php

```diff
@@ -225,4 +234,102 @@ private function getTransactionType($type)
             'class' => $class,
         ];
     }
+
+    private function getTotal()
```

Comment:
Would this calculate order total correctly? I couldn't manage to test with few transactions with different currecies.

Reply:
It should be tested with different currencies, it was designed only for EUR... 😅